### PR TITLE
Dockerhub build

### PR DESCRIPTION
**Pull request by alenkacz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,4 +24,4 @@ script:
 
 after_success:
   - test $TRAVIS_PULL_REQUEST == "false" && test "$TRAVIS_TAG" != "" && test $TRAVIS_REPO_SLUG == "alenkacz/gradle-marathon-deployer" && sudo docker run -v $(pwd):/build -v $HOME/.m2:/root/.m2 -v $HOME/.gradle:/root/.gradle -v /var/run/docker.sock:/var/run/docker.sock -e "BINTRAY_USER=$BINTRAY_USER" -e "BINTRAY_KEY=$BINTRAY_KEY" -e "GRADLE_PORTAL_KEY=$GRADLE_PORTAL_KEY" -e "GRADLE_PORTAL_SECRET=$GRADLE_PORTAL_SECRET" build-image bintrayUpload publishPlugins -Pversion="$TRAVIS_TAG" --info
-  
\ No newline at end of file
+  - ./dockerhub-push.sh
```
Comment (augi) [inline .travis.yml]: This triggers Docker Hub after every successful build, so for builds triggered by PR as well. It's not IMHO what we want.

**Review by augi — CHANGES_REQUESTED** [review]
It would be also nice to create a new README in the `docker-distribution` directory. 

I would copy Docker-related information from the main README (and also add link to the new README into the main README).

File: .travis.yml (hunk 1 above)
Comment (alenkacz) [reply .travis.yml]: good point, thanks!

**Comment by alenkacz** [comment]
@augi please re-review

File: README.md [hunk 2]
```diff
@@ -28,14 +28,7 @@ Usage
...
 
-Also [alenkacz/marathon-deployer](https://hub.docker.com/r/alenkacz/marathon-deployer/) Docker image is available so you can actually use this plugin to deploy any application from any environment where Docker is available:
-```bash
-docker run \
-    -e MARATHON_URL=http://path-to-your-marathon-instance.com \
-    -e DOCKER_IMAGE_NAME=yourorg/app:1.0.0 \
-    -v /path/to/your/marathon.json:/marathon.json
-    alenkacz/marathon-deployer
-```
+Also [alenkacz/marathon-deployer](https://hub.docker.com/r/alenkacz/marathon-deployer/) Docker image is available. See [documentation](docker-distrbution/README.md).
```
Comment (augi) [inline README.md]: Typo in the second link (missing 'i').

**Review by augi — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (alenkacz) [reply README.md]: embarassing :D I wanted to click it through before you do a review but you are so fast :D

**Comment by augi** [comment]
:+1:

**Pull request merged**